src/quoridor/QuoridorBoard.cs

```diff
@@ -0,0 +1,45 @@
+﻿using QuoridorEngine.src.solver;
+
+namespace QuoridorEngine.src.quoridor
+{
+    /// <summary>
+    /// Class representing quoridor board
+    /// </summary>
+    internal class QuoridorBoard
+    {
+        List<(int, int)> usedWalls;
+        private int width;
+        private int height;
+
+        public int Width
+        {
+            set { if (value is int and > 0) { width = value; } }
```

Comment:
Shouldn't really be allowing the dimensions to be changed after the Board is created. This would cause all sorts of bugs. Only set the dimensions once in the constructor